tests/integration/mqtt.test.js

```
@@ -1,189 +1,189 @@
-    NekoGirlStateMachine,
-    Emotion,
-    Animation,
-} = require("../../NekoGirlStateMachine");
-const { EventEmitter } = require("events");
-const mqtt = require("mqtt");
-const http = require("http");
+  NekoGirlStateMachine,
+  Emotion,
+  Animation,
+} = require('../../NekoGirlStateMachine');
+const { EventEmitter } = require('events');
```

Biome

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)